tutorials/lime_images.ipynb

```diff
@@ -116,14 +116,12 @@
    "outputs": [
```

Comment:
<p>It would be nice to get rid of the warning about Iprogress. Could you try after updating jupyter and iwidgets like is suggested in the warning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>